src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -134,6 +134,9 @@ static uint64_t window_editor_objective_options_page_hold_down_widgets[] = {
 
 class EditorObjectiveOptionsWindow final : public Window
 {
+private:
+    std::vector<RideId> _riddenRides;
```

Comment:
These aren't ridden rides these are just a list of all rides rides (what a confusing term...). Could you rename it to rideable rides (rideable rides is also a rubbish name perhaps you can think of something better) and add a comment stating `// Not shops or facilities`.

Reply:
I see, I had a misunderstanding, thank you for the explanations though I think that helps me understand more of it all! Fixing this right now.